deploy/Dockerfile

```diff
@@ -2,8 +2,12 @@ FROM api-site-base
 
 WORKDIR /docs
 
-RUN mkdir -p /docs
-ADD base/src /base/src/
+COPY ./base/src /base/src/
+
+COPY ./docs/Gemfile /docs/Gemfile
+COPY ./docs/Gemfile.lock /docs/Gemfile.lock
+RUN bundle install
```

Comment:
The site wasn't building properly without a bundle install on the docs Gemfile though it's identical to the base one??

This runs almost instantly though because it's already done the real install for the base one, so it's fairly harmless but I'm not totally sure what's going on.